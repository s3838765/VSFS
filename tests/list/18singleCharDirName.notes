NOTES V1.0
=a/
=b/
=c/
=d/
