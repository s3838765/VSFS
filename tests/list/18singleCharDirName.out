drwxrwxrwx 1 dib dib 512 Oct 22 19:54 a/
drwxrwxrwx 1 dib dib 512 Oct 22 19:54 b/
drwxrwxrwx 1 dib dib 512 Oct 22 19:54 c/
drwxrwxrwx 1 dib dib 512 Oct 22 19:54 d/
